packages/shared/src/components/notifications/InAppNotification.module.css

```diff
@@ -0,0 +1,32 @@
+.inAppNotificationAvatar {
+    img {
+        border-radius: 6px;
+        height: 2rem;
+        width: 2rem;
+    }
+
+    svg {
+        height: 1.5rem;
+        width: 1.5rem;
+    }
+}
```

Comment:
Shouldn't we use tailwind for that?

Reply:
These are inside other components, it was much easier to set them like this. But I can update the inner components to pass the size if you like.

Reply:
I see. Let's see what Lee thinks, but I don't want to block the merge. So we can ask him to review this part on the intermediate branch

Reply:
As much as possible, we would want to sway away from using `module.css`, only on certain cases where we won't be able to avoid it such as the keyframes below.

I'll note this in the other PR so we can convert it when fixing bugs/finishing touches.